src/wstool/cli_common.py

```diff
@@ -162,12 +162,18 @@ def _get_status_flags(basepath, elt_dict):
          elt_dict['actualversion'] is not None and
          elt_dict['specversion'] != elt_dict['actualversion'])):
         mflag += 'V'
-    if ('remote_revision' in elt_dict and
-        elt_dict['remote_revision'] != '' and
-        elt_dict['remote_revision'] is not None and
-        elt_dict['actualversion'] is not None and
-        'actualversion' in elt_dict and
-        elt_dict['remote_revision'] != elt_dict['actualversion']):
+    if (('remote_revision' in elt_dict and
+         elt_dict['remote_revision'] != '' and
+         elt_dict['remote_revision'] is not None and
+         elt_dict['actualversion'] is not None and
+         'actualversion' in elt_dict and
```

Comment:
well spotted. Either that, or it's always present and the check obsolete. Sadly code is too complex to tell for sure easily (my fault).